dv/verilator/pcount/cpp/ibex_pcounts.cc

```diff
@@ -32,24 +33,46 @@ const std::vector<std::string> ibex_counter_names = {
     "Multiply Wait",
     "Divide Wait"};
 
+static bool has_hpm_counter(int index) {
+  // The "cycles" and "instructions retired" counters are special and always
+  // exist.
+  if (index == 0 || index == 2)
+    return true;
+
+  // The "NONE" counter is fake. That never exists.
```

Comment:
```suggestion
  // The "NONE" counter is fake is a placeholder for the "MTIME" CSR. That never exists.
```

Reply:
Ah, that makes sense. I couldn't figure this out: how can we tell that index 1 corresponds to the MTIME CSR?

Reply:
It's in the order of MCYCLE, MTIME, MINSTRET, MHPMCOUNTER3, ...

Reply:
I'm clearly being slow here. Where does that list of CSRs come from? Is it a standard RISC-V thing? My web searches find lots of Ibex...

Reply:
Looking at e.g. the RISC-V privileged spec, I see MCYCLE, (a gap here), MINSTRET, MHPMCOUNTER3, ... in table 2.5.

Is there a spec that explains that the gap is MTIME?

Reply:
MTIME was once an actual CSR but later removed from the spec since people want it to be MMIO instead of actual CSR. But you can guess that this counter is reserved for MTIME by looking at the list of user-mode CSRs, which is CYCLE, TIME, INSTRET, HPMCOUNTER3, ...

Reply:
Ahah! Thanks for the explanation. I'll tweak this PR to match.